node-tests/helpers/compare-build.js

```diff
@@ -0,0 +1,55 @@
+var path = require('path');
+var fs = require('fs-extra');
+var walkSync = require('walk-sync');
+var execSync = require('execa').shellSync;
+
+function execute(command) {
+  console.info('Executing: ' + command);
+  execSync(command);
+}
+
+/**
+ * Builds the engine-testing application against the current upstream version
+ * of ember-engines and against the current local copy. The return value is the
+ * list of files produced by both outputs so that we can catch build differences
+ * in testing.
+ */
+module.exports = function compareBuild() {
+  // clone engine-testing
+  execute('git clone https://github.com/ember-engines/engine-testing.git');
```

Comment:
Don't need a clone:

```
wget https://github.com/ember-engines/engine-testing/archive/master.zip
unzip engine-testing-master.zip
mv engine-testing-master engine-testing
```

Then again, cross-platform means that has to happen in `node`. Gross. Nevermind, just do a shallow clone.